src/Muxlicer.hpp

```diff
@@ -0,0 +1,993 @@
+#pragma once
+#include "plugin.hpp"
+
+using simd::float_4;
+
+// an implementation of a performable, 3-stage switch, i.e. where
+// the state triggers after being dragged a certain distance
+struct BefacoSwitchMomentary : SVGSwitch {
+	BefacoSwitchMomentary() {
```

Comment:
Here I'm trying to implement a three stage momentary with the middle state as the resting state. It wasn't immediately obvious how to do this so if there is a more idiomatic way to do this then let me know!

Reply:
It's a weird mouse interaction, but what you have is fine for a hack.